bee-network/src/alias.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// A macro that creates a shorter representation of a [`PeerId`](crate::PeerId). Mostly useful for logging.
+#[macro_export]
+macro_rules! alias {
```

Comment:
I am open for suggestions as long as it is short. Let's call it `alex!` (**al**iasing **ex**treme) instead :trollface:

Reply:
Why should it be short 😅  ?

Reply:
hmm, because the crate itself uses it everywhere, and I find `peer_alias(peer_id)` too verbose. It's already clear what `alias` does when everything you pass in is called `peer_id`. I never shorten `peer_id` to `id` where `peer_alias(id)` would suddently look much better.

Reply:
Alright your call, I'm not too much of an advocate of shortening only for the sake of shortening but sure 👍

Reply:
Yea, we hit personal taste territory here. I am the last one being against changing it if more people complain or find it illogical or useless :+1: @Adam-Gleave You want to give your input on this before I close this discussion? Another idea I head was to call it `shorten!`. I think `shorten!(peer_id)` reads pretty nice, too! :smile:

Reply:
Okay I am closing this as I am getting no response. :cry: